tests/test_html_to_json.py

```diff
@@ -0,0 +1,77 @@
+import json
+import os
+from pathlib import Path
+from mbtools.html_to_json import html_to_json, main
+import pytest
+
+HTML_FRAGMENT1 = (
+    '<p dir="ltr" style="color: blue">'
+    '(6, 0)'
+    '<br>'
+    '</p>'
+    '<p style="text-align: left">'
+    'words'
+    '</p>'
+    '<p style="text-align: center">'
+    'words'
+    '</p>'
+)
+HTML_FRAGMENT2 = (
+    '<h1>Some Header</h1>'
+    '<p>Some text</p>'
+)
+
+
+@pytest.fixture
```

Comment:
This is nit-picky, but I noticed it so I'll leave a comment :joy: - Your test cases would all pass if you had a bug in your code where you wrote JSON files to `html_directory` instead of `json_directory`. You'd have to tweak your test fixture / code a bit, but probably could address that test gap pretty easily if you wanted.